cmd/build.go

```diff
@@ -52,16 +52,25 @@ func newBuildCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// buildCmd provides the ability run local builds, or setup/delete Tekton builds, for an appsody project
 	var buildCmd = &cobra.Command{
 		Use:   "build",
-		Short: "Locally build a docker image of your appsody project",
-		Long:  `This allows you to build a local Docker image from your Appsody project. Extract is run before the docker build.`,
+		Short: "Locally build a container image of your Appsody project",
+		Long: `Build a local container image of your Appsody project. The stack along with your Appsody project is first extracted to a local directory, before the docker build is run.
+
+By default, the built image is tagged with the project name you specified when first initialising your Appsody project. If you did not specify a project name at the time of initialisation, the built image will be tagged with directory name at the root of your Appsody project.
```

Comment:
Suggest small tweaks - think we can make this a bit shorter and simpler by leaving out some things:
"By default, the built image is tagged with the project name that you specified when you initialised your Appsody project. If you did not specify a name, the image is tagged with the name of the directory at the root of your Appsody project."